API.md

```diff
@@ -1275,6 +1275,69 @@ curl http://localhost:<port>/irods-http-api/<version>/query \
 }
 ```
 
+### add_specific_query
+
+Adds a specific query.
+
+#### Request
+
+```bash
+curl http://localhost:<port>/irods-http-api/<version>/query \
+    -H 'Authorization: Bearer <token>' \
+    --data-urlencode 'op=add_specific_query' \
+    --data-urlencode 'name=<string>' \ # The name of specific query.
+    --data-urlencode 'sql=<string>' # The SQL to map to the specific query name.
+```
+
+#### Response
+
+If an HTTP status code of 200 is returned, the body of the response will contain JSON. Its structure is shown below.
+
+```js
+{
+    "irods_response": {
+        "status_code": 0
+        "status_message": "string" // Optional
+    },
+    "rows": [
+        ["string", "string", "string"],
+
+        // Additional rows ...
+    ]
+}
+```
+
+### remove_specific_query
+
+Removes a specific query.
```

Comment:
Would it be worth saying "SpecificQuery" or making reference to `iadmin rsq` or something? The phrase "specific query" reads to me like "I would like to remove this particular query" rather than the iRODS concept of a "Specific Query" (which is named such for parity with a "General Query" or GenQuery). Hope that makes sense... Maybe it's clear as it is.

Reply:
Good question. Rephrasing that to use SpecificQuery or something is better. Will change.

Reply:
Done.